pages/settingspage.py

```diff
@@ -124,8 +108,17 @@ def set_data(self):
             self.buy_one_checkbox.setChecked(True)
         if settings['dont_buy']:
             self.dont_buy_checkbox.setChecked(True)
-        self.random_delay_start.setText(settings["random_delay_start"])
-        self.random_delay_stop.setText(settings["random_delay_stop"])
```

Comment:
Bug: deleting these lines broke the random delay functionality. Here's the fix:
`

        if settings['random_delay_start']:
            self.random_delay_start.setText(settings["random_delay_start"])

        if settings['random_delay_stop']:
            self.random_delay_stop.setText(settings["random_delay_stop"])`